test/WebJobs.Script.Tests/TestScripts/Node/Scenarios/index.js

```diff
@@ -18,6 +18,16 @@ var assert = require('assert');
     else if (scenario == 'promiseResolve') {
         Promise.resolve().then(() => context.done());
     }
+    else if (scenario == 'promiseApiResolves') {
```

Comment:
You see how we've been limping along with full E2E tests for our Node interop shim. +1000 extra credit points if you can put in place some Mocha tests for this stuff that run automatically on the CI machine :)